index.js

```diff
@@ -1070,17 +1086,18 @@
       } else {
         seen$ = seen;
       }
+      var expandUnknown2 = expandUnknown4 (seen$) (value);
```

Comment:
When partially applying curried functions for the sake of efficiency, I've been finding it helpful to indicate the number of arguments remaining. It's difficult to come up with unique names without such a system. :)